ci/build-ios_client.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source .shared-ci/scripts/pinned-tools.sh
+source .shared-ci/scripts/profiling.sh
+
+UNITY_PROJECT_DIR="$(pwd)/workers/unity"
+
+markStartOfBlock "$0"
+
+markStartOfBlock "Building iOSClient"
+
+TARGET=${1:-local}
+
+# Back-compat: we need to run this to generate the bridge configurations.
+spatial build build-config IosClient
+
+pushd "${UNITY_PROJECT_DIR}"
+    dotnet run -p ../../.shared-ci/tools/RunUnity/RunUnity.csproj -- \
```

Comment:
just out of curiosity: what does the output of building it via the build system look like?

Reply:
Its piped into the log file - but it looks like a normal Unity build tbh